### ISSUE TITLE
Question about Roberta-large results

### ISSUE THREAD

**Issue by nLearnERp** [zjunlp/DocuNet#28]
Hello author, we are very interested in your work, but we have not changed the code, and we cannot reproduce your results on robert-larger, and hope to get your help.
The results we have reproduced so far, the F1 on the development set is 56%.
Thanks.

**Comment by njcx-ai** [comment]
Hi, thank you for your attention. Our code was sorted out and reproduced before uploading. As posted by @zzhinlp that "The reproduced results are normal on the validation set", the results on the validation set should be fine. So we are also confused by your low results. This may also be due to the instability of training caused by the update of related dependent libraries in recent years.

**Comment by nLearnERp** [comment]
> 您好，感谢您的关注。我们的代码在上传之前已经整理并转载了。正如发布者@zzhinlp“验证集上再现的结果是正常的”，验证集上的结果应该没问题。所以我们也对你们的低结果感到困惑。这也可能是由于近几年相关依赖库的更新导致训练的不稳定。

Thank you for your answer. The update of dependent libraries may cause this kind of problem. The result on BERT is normal, but the result on robert is very strange.

**Comment by njcx-ai** [comment]
In general, it is unreasonable that the result of roberta_large is lower than that of bert_large. The high probability is that the key hyperparameters are not right in different environment configurations.

**Comment by nLearnERp** [comment]
Thank you for your answer. We understand and agree with your point of view, we will try to vary the hyperparameters to reproduce the results.
